pkg/cmd/roachtest/tests/mixed_version_job_compatibility_in_declarative_schema_changer.go

```diff
@@ -71,105 +78,120 @@ func planAndRunSchemaChange(
 	c cluster.Cluster,
 	node option.NodeListOption,
 	schemaChangeStmt string,
-) {
+) error {
```

Comment:
curious: Why do you add a `error` as the return? It seems it's not used anyway.

Reply:
this gets bubbled up to our executeSupportedDDLs function; that error is then handled by the new mixed-version framework (since we call executeSupportedDDLs in `InMixedVersion`): https://github.com/cockroachdb/cockroach/blob/fbd1bb8b3699a999a14144229428683fd7e26fe7/pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go#L161-L169